src/views/BedrijfDetailView.vue

```diff
@@ -1,98 +1,41 @@
 <template>
-  <section>
-    <utrecht-heading :level="1">Bedrijfsinformatie</utrecht-heading>
-    <nav>
-      <ul>
-        <li>
-          <router-link :to="{ name: 'bedrijven' }">{{
-            "< Bedrijven zoeken"
-          }}</router-link>
-        </li>
-      </ul>
-    </nav>
-    <simple-spinner v-if="klant.loading" />
-    <bedrijf-details v-else-if="klant.success" :klant="klant.data" />
```

Comment:
misschien scope creep, maar wat mij betreft kijken we hierbij ook een betje naar de inhoud van de tabs. De knoppen van het bedrijfdetailformulier staan nu op een vreemde plaats vind ik. En De overlap met persoondetails.vue is erg groot.  Dit zou een moment kunnen zijn om er een KlantContactgegevensForm.vue componentje uit te destileren

Reply:
Ik heb er een gedeelde component van gemaakt en de knoppen verplaatst naar beneden. Sta open voor suggesties voor een betere plek.